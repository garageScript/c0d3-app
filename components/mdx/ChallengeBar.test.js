import React from 'react'
import { render } from '@testing-library/react'
import ChallengeBar from './ChallengBar'
describe('ChallengeBar component', () => {
  test('Should created redirect based on input', () => {
    const { container } = render(<ChallengeBar src="/foo" />)
    expect(container).toMatchSnapshot()
  })
})
