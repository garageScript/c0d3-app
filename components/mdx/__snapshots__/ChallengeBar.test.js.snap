// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChallengeBar component Should created redirect based on input 1`] = `
<div>
  <div
    class="d-flex justify-content-center"
  >
    <div
      class="card shadow-sm bg-primary text-white px-4 py-2 border-0"
    >
      <h1
        class="text-center font-weight-bold align-self-center"
      />
      <p
        class="px-2 m-0 text-center"
      />
      <div
        class="text-center"
      />
    </div>
  </div>
</div>
`;
